src/interactive-window/editor-integration/types.ts

```diff
@@ -101,7 +102,7 @@ export type InteractiveCellMetadata = {
     interactiveWindowCellMarker?: string;
     interactive: {
         uristring: string;
-        line: number;
+        lineIndex: number;
```

Comment:
See above, in `IGeneratedCode` the `line` property is ` based, and this is 0 based, hence renamed to avoid confusions.